examples/plugins/ethdoc/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../../tsconfig.json",
-  "include": ["src", "profile"],
+  "include": ["src", "profile", "../../../webpack.common.js"],
```

Comment:
I think it's better to take the build output of `@remixproject/plugin-iframe` as this will be the expected entrypoint for iframes plugin. 
So you shouldn' t need any webpack.common.js file.